cloud/api/protobuf/kurtosis_backend_server_api.proto

```diff
@@ -156,13 +156,18 @@ message UpdateAddressArgs {
   string country = 8;
 }
 
-message DeleteInstanceRequest {
-  string instance_id = 1;
-  string ec2_id = 2;
+message DeleteInstancesRequest {
+  repeated string instance_ids = 1;
+
+  // must be system user to delete instances
+  string system_user_api_key = 2;
```

Comment:
I am wondering if we should rename this to `api_key` for consistency with the other methods and also when we allow the user to delete its own instance.  We can do authz as part of the method implementation.

Reply:
good idea